winpr/libwinpr/winsock/winsock.c

```diff
@@ -1191,7 +1191,7 @@ SOCKET _socket(int af, int type, int protocol)
 
 	fd = socket(af, type, protocol);
 
-	if (fd < 1)
+	if (fd < 0)
```

Comment:
`stdin` is also invalid here, needs further check.

Reply:
`0` may be returned from `socket()` if `stdin` is closed and consequently the descriptor is leaked, but it is rather theoretical...